src/pages/library/DataBrowserPreview.js

```diff
@@ -83,7 +83,7 @@ const DataBrowserPreview = ({ id }) => {
           style: { fontSize: 16, textTransform: 'none' },
           onClick: () => setViewJSON({ title: `${table}, Row ${rowIndex} - ${cellKey}`, cellData: maybeJSON })
         }, ['View JSON'])],
-        [Utils.DEFAULT, () => cellContent]
+        [Utils.DEFAULT, () => cellContent ? cellContent.toString() : '']
```

Comment:
I added the ternary as well, because our linter prefers `.toString` over `_.toString`. I wanted it to be nullsafe, so this seemed like a good approach.

Reply:
Can `?.` be used here?
```suggestion
        [Utils.DEFAULT, () => cellContent?.toString()]
```

Reply:
_.toString is null-safe, so I think it's the better option.

```
The _.toString() method is used to convert the given value to a string. An empty string is returned for null and undefined values. The sign of the value -0 is preserved.

Reply:
@cahrens I agree, but the linter actually wouldn't let me use it:
```
ESLint found too many warnings (maximum: 0).

/Users/srubenstein/Desktop/code/terra-ui/src/pages/library/DataBrowserPreview.js
  86:31  warning  Use `function toString() { [native code] }` instead of `toString`  lodash-fp/preferred-alias

```